### DISCUSSION TITLE
helm and pinning an image to certain minor releases

### DISCUSSION THREAD

**Discussion by cskinfill** [renovatebot/renovate#24012] [Request Help]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

I'd like renovatebot to manage updating a helm chart's dependencies, and allow me to pin a container image to a certain release, but allow minor releases.

For instance, if the helm chart with a dependency version 1.3, and the image is tagged with 1.4.6, I'd like to have renovatebot update both the helm chart dependency to whatever the next release is, but also update the image to 1.4.x, but NOT to 1.5.x until I explicitly update that version.

Specifically, I have local helm chart that depends on an upstream chart which will be updated regularly.  In addition, the upstream helm chart includes an image that is released frequently, but occasionally (3-4 times per year) an "LTS" release is made, and I'd like to pin all image updates to stay with the LTS release (so only update minor releases of the LTS release).

Is that possible?  If so, what do I need to do to configure that properly?

Thanks

### What you tried so far.

I haven't tried this scenario yet.

**Comment by viceice** [comment]
configure a package rule and either use allowed versions or dashboard approval for that
